packaging/osx/sharedframework/scripts/postinstall

```diff
@@ -6,8 +6,25 @@
 
 PACKAGE=$1
 INSTALL_DESTINATION=$2
+LIBCRYPTO_NAME=libcrypto.1.0.0.dylib
+LIBSSL_NAME=libssl.1.0.0.dylib
+OPENSSL_PATH=/usr/local/opt/openssl/lib
```

Comment:
You probably shouldn't set this variable if it's already set.  That way non-standard locations and/or macports users can be given the power-user instructions of "set this environment variable to where libcrypto and libssl live, then run the script"